github-issue-summary-to-email/main.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/github;
+import wso2/choreo.sendemail as email;
+
+// Github client configuration parameters
+configurable http:BearerTokenConfig gitHubTokenConfig = ?;
+configurable string repositoryName = ?;
+configurable string repositoryOwner = ?;
```

Comment:
Isn't it better to use `orgName` and `repoName` instead of `repositoryOwner` and `repositoryName`?

Reply:
Updated in [5131705](https://github.com/ballerina-guides/integration-samples/pull/35/commits/51317055927a041c645786b20bcb6af3864f1081)